src/irr/asset/CXMeshFileLoader.h

```diff
@@ -193,11 +195,11 @@ class CXMeshFileLoader : public asset::IAssetLoader
 
 	bool parseDataObjectMaterial(SContext& _ctx, video::SCPUMaterial& material);
 
-	bool parseDataObjectAnimationSet(SContext& _ctx);
+	bool parseDataObjectAnimationSet(SContext& _ctx, orientationFunction performActionBasedOnOrientationSystem);
```

Comment:
use a reference, the functional object is very expensive to pass around by copy

In all honesty I'd prefer a bunch of nice staateless static methods to a lambda

Reply:
yup, it will be fixed

Reply:
@devshgraphicsprogramming  well, how should it look like?
do you mean a static method expecting two lambdas for handling orientation?

Reply:
@devshgraphicsprogramming  I mean, if there should be such a method, it should have access to `_params`. Should I create a pointer to it as a member?

Reply:
@devshgraphicsprogramming how should I apply flipping on BAW Loader?

Reply:
are there still questions?

Reply:
it's been valid still
> how should I apply flipping on BAW Loader exactly?